ui/components/app/shield-entry-modal/shield-entry-modal.tsx

```diff
@@ -179,7 +179,7 @@ const ShieldEntryModal = ({
       autoFocus={false}
       isClosedOnOutsideClick={false}
       isClosedOnEscapeKey={false}
-      onClose={handleOnClose}
+      onClose={() => handleOnClose()}
```

Comment:
All changes in this file come from this PR https://github.com/MetaMask/metamask-extension/pull/38188, which was not included in 13.11.0, but was cherry-picked in 13.10.2.